__tests__/app.test.js

```diff
@@ -26,3 +26,65 @@ describe("/api/categories", () => {
     return request(app).get("/api/cetagories").expect(404);
   });
 });
+
+describe("/api/reviews/:review_id", () => {
+  it("GET 200 - responds with a review object containing all the appropriate properties", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body.review_id).toBe(2);
+        expect(body).toHaveProperty("title", expect.any(String));
+        expect(body).toHaveProperty("designer", expect.any(String));
+        expect(body).toHaveProperty("owner", expect.any(String));
+        expect(body).toHaveProperty("review_img_url", expect.any(String));
+        expect(body).toHaveProperty("review_body", expect.any(String));
+        expect(body).toHaveProperty("category", expect.any(String));
+        expect(body).toHaveProperty("created_at", expect.any(String));
+        expect(body).toHaveProperty("votes", expect.any(Number));
+      });
+  });
+  it("404 - responds with an error if given an ID that does not exist", () => {
+    return request(app)
+      .get("/api/reviews/9999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "ID Not Found" });
+      });
+  });
+  it("400 - responds with an error if given an ID that is not a number", () => {
+    return request(app)
+      .get("/api/reviews/notAnId")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "Invalid ID" });
+      });
+  });
+});
+
+describe("/api/reviews", () => {
+  it("GET 200 - should respond with an array of review objects with all the appropriate properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+        expect(body).toHaveLength(13);
```

Comment:
Need to be sent back with a key of `reviews`

Reply:
Need to also check for a comment_count

comment_count which is the total count of all the comments with this review_id - you should make use of queries to the database in order to achieve this.